swarm/chunk/tags.go

```diff
@@ -0,0 +1,233 @@
+package chunk
+
+import (
+	"context"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"math/rand"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+)
+
+var (
+	errExists = errors.New("already exists")
+	errNoETA  = errors.New("unable to calculate ETA")
+)
+
+// tags holds the tag infos indexed by name
+type tags struct {
```

Comment:
This type is not exported and used only in tests. Will it be exported in the future?

Reply:
yes